lib/apis/v3/ippool.go

```diff
@@ -45,6 +49,10 @@ type IPPoolSpec struct {
 	NATOutgoing bool `json:"natOutgoing,omitempty"`
 	// When disabled is true, Calico IPAM will not assign addresses from this pool.
 	Disabled bool `json:"disabled,omitempty"`
+
+	// Deprecated: this field is only used for APIv1 backwards compatibility.
+	// Setting this field is not allowed, this field is for internal use only.
+	V1IPIP *apiv1.IPIPConfiguration `json:"ipip,omitempty"`
```

Comment:
I have a sneaky feeling kubernetes likes field names to match.  I don't think we need to rename it,.